windIO/IEA-22-280-RWT_Floater.yaml

```diff
@@ -7,7 +7,7 @@ assembly:
   rotor_orientation: Upwind
   number_of_blades: 3
   hub_height: 170.0
-  rotor_diameter: 280.0
+  rotor_diameter: 282.37660
```

Comment:
how do you arrive at this diameter? 2*(4.2+136.5)=281.4. Is it computed by WISDEM?

Reply:
Thank you @fzahle for the comment. in windIO we've defined rotor diameter as hub diameter + 2*blade length. WISDEM estimates a blade length of 136.988 meters, because the reference axis is 136.5 along z, but then there is 6 meter of prebend. Was the curved length supposed to be 136.5?

Reply:
ah, you use curved length. And I see now that this is also what it states in the WindIO schema: `Diameter of the rotor, defined as two times the blade length plus the hub diameter`. I guess it's also pretty common to use the projected blade length+hub in the rotor plane.